python/practices/practice-3/src/path1/triple_double_letters.py

```diff
@@ -0,0 +1,60 @@
+"""
+Check and prints words containing three consecutive double letters
+"""
+
+
+import os
+from helpers.file_reader import read_file_content
+from constant.constant_file import word_file
+
+
+def contains_three_double(word):
+    """
+    Check if a word contains three consecutive double letters.
+
+    Parameters:
+        - word (str): The input word to check.
+
+    Returns:
+        - bool: True if the word contains three consecutive double letters,
+        False otherwise.
+    """
+
+    index = 0
+
+    while index < len(word) - 5:
+        if (
+            word[index] == word[index + 1]
+            and word[index + 2] == word[index + 3]
+            and word[index + 4] == word[index + 5]
+        ):
+            return True
+        index += 1
+
+    return False
+
+
+def find_triple_double(file_content):
+    """
+    Prints words with triple double letters from word list.
+    """
+
+    lines = file_content.split('\n')
+
+    for line in lines:
+        word = line.strip()
+
+        if contains_three_double(word):
+            print(word)
+
+
+def main():
+    """
+    Prints words containing three consecutive double letters
+    """
+
+    print(os.path.basename(__file__))
+    file_content = read_file_content(word_file)
+
+    if file_content is not None:
```

Comment:
Please review Writing Idiomatic Python 3, if statement section.

Reply:
Can you think about including this condition in find_triple_double? I think that would be more reasonable.

Reply:
Done